spec/classes/nodejs_spec.rb

```diff
@@ -18,7 +18,7 @@
       it 'the file resource root_npmrc should be in the catalog' do
         is_expected.to contain_file('root_npmrc').with(
           'ensure' => 'file',
-          'path'    => '/root/.npmrc',
+          'path'    => '/.npmrc',
```

Comment:
This is unexpected. We should fix a value for that fact. Could you add it to ```spec/default_facts.yml```? I'll submit a PR to modulesync_config to ensure it's not overridden.